crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
Since we are at it, how about declaring _asn1_mon as static and const? Yes, it's referred from a_utctm.c, but I'd say that a_utctm.c can have own static copy, it's not that big loss. The gain is better readability since you don't have to cross-reference files...